### PR TITLE
Feature/279 multithread app

### PR DESCRIPTION
**Pull request by bartimys1**
#279

### CODE REVIEW

File: HW8/Ivan_Pavlov/Multithread/Multithread.Console/Program.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    using Multithread.Console.Repo;
+    using Multithread.Core.Services;
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int endIteration = 3;
+            string url = "https://en.wikipedia.org/wiki/Mummia";
+            var linkService = new LinksServices(new LinksRepo(), new WebRepo());
```
Comment (chesdenis) [inline HW8/Ivan_Pavlov/Multithread/Multithread.Console/Program.cs]: please use DI container here

File: HW8/Ivan_Pavlov/Multithread/Multithread.Core/Services/LinksServices.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    using HtmlAgilityPack;
+    using Multithread.Core.Models;
+    using Multithread.Core.Repo;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class LinksServices
```
Comment (chesdenis) [inline HW8/Ivan_Pavlov/Multithread/Multithread.Core/Services/LinksServices.cs]: please rename to LinksService

File: HW8/Ivan_Pavlov/Multithread/Multithread.Console/Program.cs (hunk 1 above)
Comment (bartimys1) [reply HW8/Ivan_Pavlov/Multithread/Multithread.Console/Program.cs]: fix

File: HW8/Ivan_Pavlov/Multithread/Multithread.Core/Services/LinksServices.cs (hunk 2 above)
Comment (bartimys1) [reply HW8/Ivan_Pavlov/Multithread/Multithread.Core/Services/LinksServices.cs]: fix

**Pull request merged**